safe/metadata/property/date_property.py

```diff
@@ -26,10 +9,14 @@
 from safe.common.exceptions import MetadataCastError
 from safe.metadata.property import BaseProperty
 
+__copyright__ = "Copyright 2016, The InaSAFE Project"
+__license__ = "GPL version 3"
+__email__ = "info@inasafe.org"
+__revision__ = '$Format:%H$'
+
 
 class DateProperty(BaseProperty):
-    """A property that accepts date input
-    """
+    """A property that accepts date input."""
```

Comment:
If you want to be landscape compliant, you need one new line before and after class docstring ;-)